highfidelity/hifi
# Fix dpi property

The fix implemented parsing messages with properties updates from html/js world
Since QScriptEngine not capable to work in non-GUI thread its better to pass properties thru QVariant

Test plan:
- press Create button, create Web entiy
- on Properties tab scroll down to DPI settings. Change DPI.
- make sure DPI gets changed
- make sure other properties updates still works

check case https://highfidelity.fogbugz.com/f/cases/5182/Can-t-change-Resolution-DPI-on-a-web-entity
